subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/CreatePullRequest.java

```diff
@@ -37,16 +43,22 @@ public void createPullRequest(CreatePullRequestTask task, GitHubApi gitHubApi) t
         gitHubApi.post("/repos/" + task.getUpstreamRepositoryName() + "/pulls", body);
     }
 
+    private void checkPullRequestMetadata(CreatePullRequestTask task) {
+        if (isBlank(getTitle(task))) {
```

Comment:
Since CreatePullRequestTask#pullRequestDescription and CreatePullRequestTask#pullRequestTitle have @Input annotation on them, there is no need to throw an exception, Gradle will validate it for us when someone runs this task without setting these properties. You can take a look at: https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/Optional.html